### PR TITLE
Change Ur/Web default display_name

### PR DESCRIPTION
**Pull request by achlipala**
...to get more uniform naming in results visualizations.

### CODE REVIEW

File: frameworks/Ur/urweb/benchmark_config [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "framework": "urweb",
   "tests": [{
     "default": {
+      "display_name": "urweb",
```
Comment (msmith-techempower) [inline frameworks/Ur/urweb/benchmark_config]: This should be included in every test, not just `default`.

**Comment by achlipala** [comment]
Hm... but the other tests already get the names that I want, in the visualized results on the web.  Is it really still necessary to change them, too?

**Comment by msmith-techempower** [comment]
Perhaps not; I guess it might use `display_name` if available, otherwise it tries `framework`-`testname`, and if it it `default`, it uses... Platform?

Not sure.

**Comment by achlipala** [comment]
I'm happy to take my chances on this cosmetic issue for the next run of results. :)  My theory has been that this PR does accomplish my personal goal for consistent naming.

**Comment by msmith-techempower** [comment]
Well, I've already kicked off the 5th preview and we are still aiming for the final run to start on Friday, so if I merge this in, it will likely be used on the final run and nothing else (just because of the timing). Either way, we will try and make sure that all of the displayed values are `urweb`.

**Comment by achlipala** [comment]
OK.  I'm happy with any result that leads to consistent naming.  Thanks.

**Pull request merged**